problem_builder/dashboard.py

```diff
@@ -402,6 +403,7 @@ def student_view(self, context=None):  # pylint: disable=unused-argument
                 block['mcqs'].append({
                     "display_name": mcq_block.display_name_with_default,
                     "value": value,
+                    "accessible_value": _("Score: {value}").format(value=value) if value else _("No value yet"),
```

Comment:
perhaps "No score yet"? "value" isn't as specific ("value for what?") whereas "score" or "problem score" is much more specific.